### DISCUSSION TITLE
I really want to submit my modifications, but I don't know how to operate it.

### DISCUSSION THREAD

**Discussion by HelloLaohe** [lllyasviel/Fooocus#1868] [Ideas]
![style](https://github.com/lllyasviel/Fooocus/assets/131928833/45bbe6f5-1662-4b76-be9c-d7fd6b2e2c4f)
 I haven't been involved in development for over ten years, but I really like Fooocus. I have developed many features for github.com, but I don't know how to submit them. Meanwhile, I can only use it as an extension script for Tampermonkey.

**Comment by HelloLaohe** [comment]
Recently, I've seen the mouse hover effect, but it's really inconvenient. I hope someone who understands submission can help me. I can contribute my own code.

**Comment by eddyizm** [comment]
You should make a fork and add your script there to start.

**Comment by thiner** [comment]
Start from here:
![image](https://github.com/lllyasviel/Fooocus/assets/1897227/dbc69355-24e0-409f-be69-c8831d81a6e5)

This button will guide you to create folk repo in your namespace, then you can submit your code changes to the folk repo. When you finish the code changes, you can create a "pull request" merge the changes back to Fooocus repo.

**Comment by HelloLaohe** [comment]
thank you very much.
